.travis.yml

```diff
@@ -8,4 +8,20 @@ sudo: required
 services:
   - docker
 
-script: hack/ci-check.sh
+script:
+- export BRANCH=$(if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then echo $TRAVIS_BRANCH; else echo $TRAVIS_PULL_REQUEST_BRANCH; fi)
```

Comment:
ouch https://github.com/travis-ci/travis-ci/issues/6652 :(

Reply:
Yeah, this isn't great, and I borrowed this line from Sonobouy. If we have to do this logic, I prefer we do it once and the rest of the scripts can assume it's there.